docs/source/tutorial.rst

```diff
@@ -42,14 +42,32 @@ To use a loader, (for example, ``urbansound8k``) you need to initialize it by ca
 You can indicate where the data would be stored and access by passing a path to ``data_home``, as explained below. Now ``us8k`` is a ``Dataset``
 object containing common methods, described in the following.
 
+Some datasets have more that one version available. Soundata can handle more than one version if included in the dataloader.
```

Comment:
```suggestion
Some datasets have more than one version available. Soundata can handle more than one version if included in the dataloader.
```

BTW, I would highly recommend to use semantic linefeeds in the future, when writing documentation (sorry, can only share as an image):

![image](https://github.com/soundata/soundata/assets/173624/99648f70-b011-4fa6-abfe-b6a5d96d7b5b)

Reply:
I see. Indeed, thanks for the suggestion. Actually, longer lines make editing more difficult as well, since you cannot read the entire thing and you have to move left and right a lot. Will take into account for sure!